contentcuration/contentcuration/frontend/shared/data/applyRemoteChanges.js

```diff
@@ -186,10 +186,17 @@ class ReturnedChanges extends ChangeDispatcher {
     }
 
     // Publish changes associate with the channel, but we open a transaction on contentnode
-    return transaction(change, TABLE_NAMES.CONTENTNODE, () => {
+    return transaction(change, TABLE_NAMES.CONTENTNODE, TABLE_NAMES.CHANGES_TABLE, () => {
       return db
         .table(TABLE_NAMES.CONTENTNODE)
         .where({ channel_id: change.channel_id })
+        .and(node => {
+          const unpublishedNodeIds = db[TABLE_NAMES.CHANGES_TABLE]
+            .where({ table: TABLE_NAMES.CONTENTNODE, key: node.id })
+            .limit(1)
+            .toArray();
+          return unpublishedNodeIds.length === 0;
+        })
```

Comment:
@rtibbles, we should be good now, I think?

Reply:
I think so!